ldclient.go

```diff
@@ -93,43 +91,45 @@ func MakeCustomClient(apiKey string, config Config, waitFor time.Duration) (*LDC
 		config.PollInterval = 1 * time.Second
 	}
 
-	requestor := newRequestor(apiKey, config)
```

Comment:
I moved things around in this initialization code so it flows better for offline/LDD mode.